deployments/kubehound/graph/Dockerfile

```diff
@@ -59,9 +59,22 @@ ENV gremlinserver.metrics.slf4jReporter.enabled=false
 ENV gremlinserver.metrics.graphiteReporter.enabled=false
 ENV gremlinserver.metrics.csvReporter.enabled=false
 
+# Add safety net settings to prevent OOM and other issues
+ENV janusgraph.query.force-index=true
+ENV janusgraph.cluster.max-partitions=512
+ENV janusgraph.query.batch=true
+ENV janusgraph.query.hard-max-limit=10000
+ENV janusgraph.query.smart-limit=true
+
+# Enable cache
+ENV janusgraph.cache.db-cache=true
+ENV janusgraph.cache.db-cache-clean-wait=20
+ENV janusgraph.cache.db-cache-time=180000
+ENV janusgraph.cache.db-cache-size=0.5
+
 # Performance tweaks based on: https://www.sailpoint.com/blog/souping-up-the-gremlin/
 # gremlinPool will default to Runtime.availableProcessors()
-ENV gremlinserver.gremlinPool=0 
+ENV gremlinserver.gremlinPool=8
```

Comment:
Is there a reason not to use the default of `Runtime.availableProcessors()`?

Reply:
`Runtime.availableProcessors()` has issues when running into a docker container.

Reply:
I will update the jvmopts to enable JAVA_OPTS - `-XX:+UseContainerSupport` to ensure support.